nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/test/resources/log4j.properties

```diff
@@ -13,8 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 log4j.rootLogger=INFO, CONSOLE
```

Comment:
Is this file necessary?  Log4j calls should be routed to SLF4J and this configuration file should be removed.

Reply:
It's indeed not. Thanks, I'll remove it.